app/src/main/java/com/pileproject/drive/util/fragment/AlertDialogFragment.java

```diff
@@ -39,16 +41,6 @@
  * show a dialog.
  */
 public class AlertDialogFragment extends DialogFragment {
-    private static final String KEY_TITLE          = "title";
```

Comment:
I don't agree that private keys are in XMLs, since
* they are not referred by other classes
* it is hard to write down the long ''getString(R.string....)''